_data/toc.yaml

```diff
@@ -24,6 +24,8 @@ guides:
     title: Docker for Mac
   - path: /docker-for-windows/install/
     title: Docker for Windows
+  - path: /docker-ee-for-windows/install/
+    title: Docker EE for Windows Server 2016   
```

Comment:
I wonder if we should leave off the `2016` in the TOC text, and just include it in the page title itself.